runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -166,13 +171,15 @@ public void onChatMessage(ChatMessage event)
 				//set it to 17 because this message is triggered first before the above chat event
 				bindNeckOverlay.bindingCharges = 17;
 			}
+			return;
 		}
 		if (config.degradingNotification())
 		{
 			if (event.getMessage().contains("Your pouch has decayed through use."))
 			{
 				notifier.notify(POUCH_DECAYED_MESSAGE);
 			}
+			return;
```

Comment:
Same as other comment, this should be within the pouch decayed `if` block, after the `notifier.notify()` call.